### DISCUSSION TITLE
Remove subtitle after extraction

### DISCUSSION THREAD

**Discussion by gabelul** [YaoFANGUK/video-subtitle-extractor#142] [Q&A] [answered]
Was wondering if there's a way to remove the subtitle after extraction. Something like remsubtitles that deletes the masked-out areas from the input frames and blends in the surrounding colors from ttps://github.com/roybaer/burnt-in-subtitle-extractor.

thanks

**Comment by YaoFANGUK** [comment]
this is a difficult task

**Reply by knreddypc** [reply]
There is plug in for Virtualdub called MSU Subtitle Remover 3.0b2 , it works more than 50 % accuracy on subtitle background. There is an excellent program on Sourceforge Video Subtitle extractor which extracts hardcoded subtitles accurately , that means it knows the location of each character.  now only merging with surround  colour is required.

**Comment by YaoFANGUK** [comment, marked as answer]
now it is available at: https://github.com/YaoFANGUK/video-subtitle-remover
